example/ExampleOptions.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Example;
+
+use Equip\Queue\AbstractOptions;
+
+class ExampleOptions extends AbstractOptions
```

Comment:
This makes it super easy to convert any existing v2 of `equip/command` command to be used as a background job.